openshift-install-powervs

```diff
@@ -609,6 +610,13 @@ function precheck_input {
     VAR_PUBLIC_KEY_FILE=$(read_varfile "public_key_file")
     VAR_RHEL_SUBS_USER=$(read_varfile "rhel_subscription_username")
   fi
+  
+  for var in $vars; do 
+    if [[ $var == *"compute-vars"* ]]; then
+      varfile="var.tfvars"
+      vars=" -var-file ../$varfile -var-file ../automation/compute-vars/$flavor.tfvars" 
```

Comment:
`vars` is already assigned some line above in original code. You just need to append the compute-vars file from the the -flavor value here.